multi-arch-builders/provisioning/aarch64/networks.tf

```
@@ -0,0 +1,51 @@
+resource "aws_vpc" "vpc" {
+  count = var.distro == "fcos" ? 1 : 0
+  cidr_block           = "172.31.0.0/16"
+  tags = {
+    Name = "${var.project}-vpc"
+  }
+}
+
+resource "aws_internet_gateway" "gw" {
+  count = var.distro == "fcos" ? 1 : 0
+  vpc_id   = aws_vpc.vpc[0].id
+}
+
+data "aws_availability_zones" "azs" {
+  state    = "available"
+}
+
+variable "private_subnet_cidrs" {
+ type        = list(string)
+ description = "Private Subnet CIDR values"
+ default     = ["172.31.1.0/24", "172.31.2.0/24", "172.31.3.0/24", "172.31.4.0/24", "172.31.5.0/24", "172.31.6.0/24", "172.31.7.0/24", "172.31.8.0/24"]
+}
+
+resource "aws_subnet" "private_subnets" {
+ count      = length(data.aws_availability_zones.azs.names) * (var.distro == "fcos" ? 1 : 0)
```

Optional: This might be clearer:

```suggestion
 count      = var.distro == "fcos" ? length(data.aws_availability_zones.azs.names) : 0
```